lib/simctl.ts

```diff
@@ -117,7 +117,9 @@ export class Simctl implements ISimctl {
 	}
 
 	private simctlExec(command: string, args: string[], opts?: any): any {
-		let fullCommand = (["xcrun", "simctl", command].concat(args)).join(" ");
+		let escapedArgs = args.map((arg) => `"${arg}"`);
```

Comment:
Sure, great catch! Didn't know the difference between exec and spawn.
I've changed it the way you requested, but have some concerns - I've imported the node's built-in `child_process` module in order to invoke `spawnSync`. I'm not sure what happens to the errors in this case and should we create a `spawnSync` in our own child-process module? @Mitko-Kerezov @rosen-vladimirov